.github/ISSUE_TEMPLATE/kubernetes_update.md

```diff
@@ -39,10 +39,9 @@ List of items to do for upgrading to {version_1} {version_2} {version_3}
         - [ ] `dev_minikube_example_cluster_version`, which is {version_2} with the supported patch version
 - [ ] If client-go pulled in a new version of gRPC, then also
     - [ ] Update the `grpc_release_tag` in the SDK [base image grpc version](https://github.com/googleforgames/agones/blob/main/build/includes/sdk.mk).
-    - [ ] Update the gRPC version number in C++ CMake scripts, `AGONES_GRPC_VERSION` [here](https://github.com/googleforgames/agones/blob/main/sdks/cpp/CMakeLists.txt)
-          and `gRPC_GIT_TAG` [here](https://github.com/googleforgames/agones/blob/main/sdks/cpp/cmake/prerequisites.cmake)
+    - [ ] Update the gRPC version number in C++ gRPC Dependency documentation [here](https://github.com/googleforgames/agones/blob/release-1.38.0/site/content/en/docs/Guides/Client%20SDKs/cpp.md).
```

Comment:
The version number is hardcoded in `examples/cpp-simple/Dockerfile` as well now. Maybe make that an arg in the docker file near the top to make it easy to update via the checklist?

Reply:
I thought about this - but then you will end up with another hardcoded value in the `Makefile` - so either way, you end up with a hardcoded value. I erred on the side of readability for the end user when looking at the example, rather than the user having to parse an `ARG` in the Dockerfile and then winding back to wherever that was specified.

Reply:
Blarg. I swore I did that! On it.